test/preview/components/retrievers/test_memory_retriever.py

```diff
@@ -0,0 +1,125 @@
+from typing import Dict, Any, List
+
+import pytest
+
+from haystack.preview import Pipeline
+from haystack.preview.components.retrievers.memory import MemoryRetriever
+from haystack.preview.dataclasses import Document
+from haystack.preview.document_stores import MemoryDocumentStore
+
+from test.preview.components.base import BaseTestComponent
+
+
+@pytest.fixture()
+def test_memory_retriever_docs():
```

Comment:
Let's not name fixtures with the `test_` prefix, that should be meant only for tests. I'd name this `mock_docs` to be fair, there's also no need to specify they're for the memory retriever since we're defining them here.